FrontendLiveReloadIT::liveReloadOnTouchedFrontendFile fails with NPE

### Description of the bug / feature

FrontendLiveReloadIT::liveReloadOnTouchedFrontendFile fails for Flow 6,7,8 with:

```
java.lang.RuntimeException: liveReloadOnTouchedFrontendFile[ANY_Chrome_](com.vaadin.flow.uitest.ui.FrontendLiveReloadIT): run failed 2 times
	at com.vaadin.testbench.RetryRule$1.evaluate(RetryRule.java:78)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:410)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at com.vaadin.flow.uitest.ui.FrontendLiveReloadIT.liveReloadOnTouchedFrontendFile(FrontendLiveReloadIT.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at com.vaadin.testbench.parallel.ParallelRunner$TBMethod.invokeExplosively(ParallelRunner.java:490)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at com.vaadin.testbench.parallel.ParallelRunner$1.evaluate(ParallelRunner.java:464)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at com.vaadin.testbench.RetryRule$1.evaluate(RetryRule.java:68)
	... 12 more
------- Stderr: -------
Jun 10, 2021 8:20:58 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
Jun 10, 2021 8:21:00 AM com.vaadin.testbench.ScreenshotOnFailureRule failed
INFO: Error screenshot written to: /opt/agent/work/a69251da2b2bb9e9/flow-tests/test-live-reload/error-screenshots/liveReloadOnTouchedFrontendFile[ANY_Chrome_](com.vaadin.flow.uitest.ui.FrontendLiveReloadIT).png
Jun 10, 2021 8:21:01 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
Jun 10, 2021 8:21:02 AM com.vaadin.testbench.ScreenshotOnFailureRule failed
INFO: Error screenshot written to: /opt/agent/work/a69251da2b2bb9e9/flow-tests/test-live-reload/error-screenshots/liveReloadOnTouchedFrontendFile[ANY_Chrome_](com.vaadin.flow.uitest.ui.FrontendLiveReloadIT).png
```

[Flow_Public_Flow_Pull_Requests_Run_Validation_17874.log](https://github.com/vaadin/flow/files/6630870/Flow_Public_Flow_Pull_Requests_Run_Validation_17874.log)

![Screenshot 2021-06-10 at 14 17 26](https://user-images.githubusercontent.com/61410877/121516201-a27a6480-c9f6-11eb-89e5-4c559f19125d.png)

--- a/flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/FrontendLiveReloadIT.java
+++ b/flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/FrontendLiveReloadIT.java
@@ -18,7 +18,6 @@ package com.vaadin.flow.uitest.ui;
 import net.jcip.annotations.NotThreadSafe;
 import org.junit.After;
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
@@ -31,7 +30,6 @@ public class FrontendLiveReloadIT extends AbstractLiveReloadIT {
         executeScript("fetch('/context/view/reset_frontend')");
     }
 
-    @Ignore("https://github.com/vaadin/flow/issues/11210")
     @Test
     public void liveReloadOnTouchedFrontendFile() {
         open();
@@ -39,7 +37,7 @@ public class FrontendLiveReloadIT extends AbstractLiveReloadIT {
         // when: the frontend code is updated
         WebElement codeField = findElement(
                 By.id(FrontendLiveReloadView.FRONTEND_CODE_TEXT));
-        String oldCode = codeField.getAttribute("value");
+        String oldCode = getValue(codeField);
         String newCode = oldCode.replace("Custom component contents",
                 "Updated component contents");
         codeField.clear();
@@ -70,7 +68,7 @@ public class FrontendLiveReloadIT extends AbstractLiveReloadIT {
         // when: a webpack error occurs during frontend file edit
         WebElement codeField = findElement(
                 By.id(FrontendLiveReloadView.FRONTEND_CODE_TEXT));
-        String oldCode = codeField.getAttribute("value");
+        String oldCode = getValue(codeField);
         String erroneousCode = "{" + oldCode;
         codeField.clear();
         codeField.sendKeys(erroneousCode); // illegal TS
@@ -86,5 +84,11 @@ public class FrontendLiveReloadIT extends AbstractLiveReloadIT {
 
         // then: the error box is not shown and the view is reloaded
         waitForElementNotPresent(By.className("v-system-error"));
+    }
+
+    private String getValue(WebElement element) {
+        Object result = getCommandExecutor()
+                .executeScript("return arguments[0].value;", element);
+        return result == null ? "" : result.toString();
     }
 }

